content/articles/one-way-anova-excel-testing-normality-homogeneity-using-python/index.md

```diff
@@ -0,0 +1,154 @@
+```
+
+After installing these dependencies, we'll proceed to create an ANOVA table with Excel.
+
+### Getting started
+#### Creating an ANOVA table with Excel
+Microsoft Excels's Analysis Toolpak add-in is a game-changer in statistical Analysis. We'll show you how to create an ANOVA table containing the sum of squares, degree of freedom, mean squares, F value, P-value, F critical.
+
+To perform a one way ANOVA, implement the following step.
+- Import your data set in any preferred Excel format.
+
+![data set](/engineering-education/Solving-a-one-way-anova-with-Excel-and-testing-for-normality-and-homogeneity-using-python/data-set.jpg)
+
+- Go to the Data tab, click on the Data Analysis sub-tab. if you can't find the sub-tab, click [here](###analysis-toolpak).
+
+- Select ANOVA: single factor and click ok.
+- Click on the input range and highlight the dataset you want to use.
+
+![input range](/engineering-education/Solving-a-one-way-anova-with-Excel-and-testing-for-normality-and-homogeneity-using-python/input-range.jpg)
+
+>Note: Highlight the numbers in the data set alone.
+
+- You can decide if you want to view it in the same spreadsheet or another spreadsheet.
+
+![result](/engineering-education/Solving-a-one-way-anova-with-Excel-and-testing-for-normality-and-homogeneity-using-python/result.jpg)
+
+In our ANOVA table above, we analysed the sum of squares and other values of the ANOVA. With this, we can solve a one way ANOVA using Microsoft Excel.
+
+#### Analysis toolpak
+The analysis toolpak is an essential add-in for what we are planning to do. To load the add-in, we'll do the following:
+- Click the file button or the Microsoft logo on the top left corner.
+- Select Excel options and scroll down to add-ins.
+- Out of the list, select Analysis Toolpak and click `Go`.
+- Check on Analysis Toolpak and select `Ok`.
+- Return to the Data tab, and you'll notice the data analysis sub-tab.
+
+#### Testing for normality using box plots
+Now we will create a Python file and import pandas, matplotlib and seaborn into our code. Pandas, Matplotlib & Seaborn are Python libraries for data analysis. 
+
+Pandas is used to import a wide variety of data formats, while Matplotlib and Seaborn are visualisation libraries used to generate powerful visualisations. Seaborn is a library that compiles statistical graphics in Python.
+
+It uses the matplotlib library. Next, we'll read our dataset (it will be in CSV file format). Then we will reshape the data frame suitable for the statistical model package and replace the column names. Now we'll generate a boxplot to see the data distribution by treatments.
+
+>Note: When reading from a directory, use double backslash or put `r` at the front of the directory address encased in a quote symbol.
+
+![unicode error](/engineering-education/Solving-a-one-way-anova-with-Excel-and-testing-for-normality-and-homogeneity-using-python/unicode-error.jpg)
+
+Below is an implementation of the above instructions:
+```Python
```

Comment:
Please change the 'P' in Python to be in small letters. Also, change it in the code snippet below.

Reply:
A content moderator @lalith1403 told me that I should make the first p in Python capital.